AndresC-AuthMethodsPolicy-SMS (#11445)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/smsauthenticationmethodtarget.md

---
title: "smsAuthenticationMethodTarget resource type"
description: "A collection of users or groups enabled to use Text Message authentication methods policy."
author: "mmcla"
localization_priority: Normal
ms.prod: "microsoft-identity-platform"
doc_type: resourcePageType
---

# smsAuthenticationMethodTarget resource type
Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

A collection of users or groups enabled to use [Text Message authentication methods policy](../resources/smsAuthenticationMethodConfiguration.md) in Azure AD.

## Properties
|Property|Type|Description|
|:---|:---|:---|
|id|String|Object ID of an Azure AD user or group.|

> |isRegistrationRequired|Boolean|Determines whether the user is enforced to register the authentication method. *Not supported*.|

Reviewer: ```suggestion
|isRegistrationRequired|Boolean|Determines whether the user is enforced to register the authentication method. **Not supported**.|
```

Author: done

> |isUsableForSignIn|Boolean|Determines if the users or groups can use this authentication method to sign in to Azure AD. *Must always be True*|

Reviewer: ```suggestion
|isUsableForSignIn|Boolean|Determines if the users or groups can use this authentication method to sign in to Azure AD. The value is always `true`.|
```

Author: done
